### ISSUE TITLE
flake8 configuration file on Mac OS X

### ISSUE THREAD

**Issue by asottile** [PyCQA/flake8#76]
In GitLab by @HongxuChen on Feb 11, 2015, 23:51

flake8's configuration file ~/.config/flake8 does not work on my Mac OS X although it works on another Debian system. The results of `flake8 --version` are as follows:

```shell
2.2.3 (pep8: 1.5.7, pyflakes: 0.8.1, mccabe: 0.2.1, naming: 0.2.2) CPython 2.7.8 on Linux
2.3.0 (pep8: 1.6.1, pyflakes: 0.8.1, mccabe: 0.3, naming: 0.2.2) CPython 2.7.6 on Darwin
```

A similar issue was found on [stackoverflow](http://stackoverflow.com/questions/28436382/flake8-not-picking-up-config-file).

As I don't like to specify the extra args each time for different IDEs, is there such a default config file location for Mac OS X?

**Comment by asottile** [comment]
In GitLab by @sigmavirus24 on Feb 12, 2015, 05:36

Have you tried downgrading pep8 on your OSX box? This seems like a potential regression in pep8

**Comment by asottile** [comment]
In GitLab by @rouge8 on Feb 13, 2015, 13:55

It might be a pep8 regression, I have an `ignore` section in my `tox.ini` that's not being respected either.

**Comment by asottile** [comment]
In GitLab by @HongxuChen on Feb 14, 2015, 24:46

Exactly as you said. when I downgrade pep8 to 1.5.7 it works fine.

**Comment by asottile** [comment]
In GitLab by @sigmavirus24 on Feb 14, 2015, 10:40

So I'm going to file an issue upstream because this was the result of a different bug being fixed and it seems to have broken way more than it fixed.

**Comment by asottile** [comment]
In GitLab by @IanLee1521 on Feb 15, 2015, 14:10

This was caused by a regression in pep8 1.6.1 and is resolved in the just released 1.6.2 version.

**Comment by asottile** [comment]
In GitLab by @sigmavirus24 on Mar 17, 2015, 22:24

See also https://mail.python.org/pipermail/code-quality/2015-March/000515.html
